tools/qc.wdl

```diff
@@ -49,23 +49,23 @@ task parse_infer_experiment {
     }
```

Comment:
Can we look at this and maybe remove any of the references to `rseqc` tools since we don't use that anymore? There may also be a docker container, not sure. This doesn't have to happen for this PR, just something I don't want to forget to clean up.

Reply:
Just went ahead and removed all references to `rseqc`